integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/CarbonScanRDD.scala

```diff
@@ -116,8 +116,8 @@ class CarbonScanRDD(
           i += 1
           result.add(partition)
         }
-      } else if (sparkContext.getConf.contains("spark.carbon.custom.distribution") &&
-                 sparkContext.getConf.getBoolean("spark.carbon.custom.distribution", false)) {
+      } else if (java.lang.Boolean
+        .parseBoolean(CarbonProperties.getInstance().getProperty("carbon.block.distribution"))) {
```

Comment:
fixed review comments
changed property to carbon.custom.block.distribution
added into CarbonCommonConstants